CMakeLists.txt

```diff
@@ -11,11 +12,73 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 include(GNUInstallDirs)
 
-add_subdirectory(pass)
-add_subdirectory(runtime)
+
+### LLVMStabilizer.so
+add_library(LLVMStabilizer MODULE)
+
+find_package(LLVM ${STABILIZER_LLVM_VERSION} REQUIRED CONFIG)
+
+target_include_directories(LLVMStabilizer PRIVATE ${LLVM_INCLUDE_DIRS})
+
+separate_arguments(LLVM_DEFINITIONS_LIST NATIVE_COMMAND ${LLVM_DEFINITIONS})
+target_compile_definitions(LLVMStabilizer PRIVATE ${LLVM_DEFINITIONS_LIST})
+
+target_sources(LLVMStabilizer
+  PRIVATE
+    pass/IntrinsicLibcalls.h
+    pass/LowerIntrinsics.cpp
+    pass/Stabilizer.cpp
+)
+
+### libstabilizer.so
+add_library(stabilizer MODULE)
+
+target_include_directories(stabilizer
+  PRIVATE
+    ${CMAKE_CURRENT_SOURCE_DIR}/Heap-Layers
+    ${CMAKE_CURRENT_SOURCE_DIR}/DieHard/src/include
+    ${CMAKE_CURRENT_SOURCE_DIR}/DieHard/src/include/math
+    ${CMAKE_CURRENT_SOURCE_DIR}/DieHard/src/include/rng
+    ${CMAKE_CURRENT_SOURCE_DIR}/DieHard/src/include/static
+    ${CMAKE_CURRENT_SOURCE_DIR}/DieHard/src/include/util
```

Comment:
I believe this paths are relative to `CMAKE_CURRENT_SOURCE_DIR` by default, so it is redundant after the move.

Reply:
Yeah, no need for it here.